docs/source/applications.rst

```diff
@@ -1,9 +0,0 @@
-Applications
```

Comment:
This actually should probably link to a read the docs for volttron-applications....instead of deleting...that doesn't exist though so this works :)